i18n/src/main/res/values/feat_setting.xml

```diff
@@ -78,6 +78,9 @@
     <string name="feat_setting_canvas_apply">apply</string>
     <string name="feat_setting_canvas_reset">reset</string>
 
+    <string name="feat_setting_epg_clock_mode">12h clock mode</string>
+    <string name="epg_clock_mode_description">12h clock</string>
```

Comment:
Tell user this preference is used to epg in the description string, Or add "EPG" prefix in the title string, and remove the description. (description isnot necessary

Reply:
Remove the description. tell user it is for epg in title.